stan/math/prim/scal/prob/beta_proportion_lccdf.hpp

```diff
@@ -0,0 +1,138 @@
+#ifndef STAN_MATH_PRIM_SCAL_PROB_BETA_PROPORTION_LCCDF_HPP
+#define STAN_MATH_PRIM_SCAL_PROB_BETA_PROPORTION_LCCDF_HPP
+
+#include <stan/math/prim/scal/meta/is_constant_struct.hpp>
+#include <stan/math/prim/scal/meta/partials_return_type.hpp>
+#include <stan/math/prim/scal/meta/operands_and_partials.hpp>
+#include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
+#include <stan/math/prim/scal/err/check_less_or_equal.hpp>
+#include <stan/math/prim/scal/err/check_nonnegative.hpp>
+#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/err/check_positive_finite.hpp>
+#include <stan/math/prim/scal/fun/size_zero.hpp>
+#include <stan/math/prim/scal/fun/value_of.hpp>
+#include <stan/math/prim/scal/fun/digamma.hpp>
+#include <stan/math/prim/scal/fun/lbeta.hpp>
+#include <stan/math/prim/scal/meta/contains_nonconstant_struct.hpp>
+#include <stan/math/prim/scal/meta/scalar_seq_view.hpp>
+#include <stan/math/prim/scal/meta/VectorBuilder.hpp>
+#include <stan/math/prim/scal/fun/constants.hpp>
+#include <stan/math/prim/scal/meta/include_summand.hpp>
+#include <stan/math/prim/scal/fun/grad_reg_inc_beta.hpp>
+#include <stan/math/prim/scal/fun/inc_beta.hpp>
+#include <cmath>
+
+namespace stan {
+namespace math {
+
+/**
+ * Returns the beta log complementary cumulative distribution function
+ * for the given probability and parameterized with location and
+ * precision parameters. Given matching containers returns the log sum
+ * of probabilities.
+ *
+ * @tparam T_y type of probability parameter
+ * @tparam T_loc type of location parameter
+ * @tparam T_prec type of precision parameter
+ * @param y probability parameter
+ * @param p location parameter
+ * @param c precision parameter
+ * @return log probability or log sum of probabilities
+ * @throw std::domain_error if p is outside (0, 1)
```

Comment:
[optional]
I'm not 100% sure this will work---did you look at doxygen output for `std::domain_error`?  I've usually just enumerated errors under a single `@throw std::domain_error`.

Reply:
I'm seeing what I expect in the doxygen output HTML files:

```
Exceptions
    std::domain_error   if p is outside (0, 1)
```
I must be missing something.